src/ui/starter-select-ui-handler.ts

```diff
@@ -957,15 +981,19 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
         }
       ],};
 
+    const passiveUnlocked = this.isPassiveUnlocked(species.speciesId);
     const passiveAvailable = this.isPassiveAvailable(species.speciesId);
-    // 'Only Passives' mode
+    const valueReductionMaxed = this.isValueReductionMaxed(species.speciesId);
+    const valueReductionAvailable = this.isValueReductionAvailable(species.speciesId);
```

Comment:
```suggestion
    const isValueReductionAvailable = this.isValueReductionAvailable(species.speciesId);
```
`is` (or `has`) prefix helps to quickly distinguish a boolean